application/controllers/FailedEmailController.php

```diff
@@ -96,7 +96,7 @@ public function actionResend()
         $deleteAfterResend = App()->request->getParam('deleteAfterResend');
         $preserveResend = is_null($deleteAfterResend);
         $item = [App()->request->getParam('item')];
-        $items = json_decode(App()->request->getParam('sItems'));
+        $items = json_decode((string) App()->request->getParam('sItems'));
```

Comment:
A little bit sloppy? getParam() already has a second argument to give default value at null.